EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+ title: Predictable Proof-of-Work (POW) Sunsetting
+author: Query0x (@Query0x)
+discussions-to: https://ethereum-magicians.org/t/eip-3374-predictable-proof-of-work-sunsetting
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-13
+---
+
+## Simple Summary
+Sets block reward to 3 and reduces it to 1 linearly over the course of about 1 year.
+
+## Abstract
+Sets the block reward to 3 ETH and then incrementally decreases it every block for 2,362,000 blocks (approximately 1 year) until it reaches 1 ETH.
+
+## Motivation
+Unnecessarily abrupt changes to the Ethereum ecosystem cause disruption and disharmony resulting in the disenfranchisement of community members while undermining stability and confidence.  While moves from Proof-of-Work to Proof-of-Stake will undoubtedly cause friction between those community members vested in either, all benefit from a measured, predictable transition.
+
+This proposal:
+
+1) Is issuance neutral.  There is no overall increase in mining rewards.
+2) Smooths the immediate impact of [EIP-1559](./eip-1559) on mining economics reducing the likeihood of reactionary responses;
+3) Introduces an ongoing, predictable reduction in future mining rewards down to 1, effectively "sunsetting" POW and codifying the move to POS;
+4) Removes economic incentives for continued development of ASICs;
+5) Allows the impacts of decreasing miner rewards to be measured and monitored rather than relying on conjecture and game theory, so adjustments can be made if necessary.
+
+
+## Specification
+Adjust block, uncle, and nephew rewards
+### Constants
+* `TRANSITION_START_BLOCK_NUMBER: TBD`
+* `TRANSITION_DURATION: 2_362_000` // (about one year)
+* `TRANSITION_END_BLOCK_NUMBER: FORK_BLOCK_NUMBER + TRANSITION_DURATION`
+* `STARTING_REWARD: 3_000_000_000_000_000_000`
+* `ENDING_REWARD: 1_000_000_000_000_000_000`
+* `REWARD_DELTA: STARTING_REWARD - ENDING_REWARD`
+### Block Reward
+```py
+if block.number >= TRANSITION_END_BLOCK_NUMBER:
+    block_reward = ENDING_REWARD
+elif block.number >= TRANSITION_START_BLOCK_NUMBER:
+    block_reward = STARTING_REWARD - REWARD_DELTA * TRANSITION_DURATION / (block.number - TRANSITION_START_BLOCK_NUMBER)
+```
+
+## Rationale
+This proposal smooths the effect of EIP-1559 on mining economics, without affecting the fee burn or increasing total ETH issuance, and mitigates the risks and uncertainties that sudden changes to mining economics impart on network security.  Picking starting and ending block reward values that are equidistant from the current block reward rate of 2 ensures the impact of this EIP will be issuance neutral over the one year time frame.  Temporarily raising the block reward to 3 blunts the initial impact of EIP-1559 and the continual reductions thereafter codify Ethereum's move to POS by increasingly disincentivizing POW.  Importantly, this approach moderates the rate of change so impacts and threats can be measured and monitored.
```

Comment:
```suggestion
Picking starting and ending block reward values that are equidistant from the current block reward rate of 2 ensures the impact of this EIP will be issuance neutral over the one year time frame.  Temporarily raising the block reward to 3 blunts the initial impact of a sudden miner revenue decrease and the continual reductions thereafter codify Ethereum's move to POS by increasingly disincentivizing POW.  Importantly, this approach moderates the rate of change so impacts and threats can be measured and monitored.
```
The first sentence is not appropriate for the Rationale, it should be part of Motivation.  That being said, as mentioned in the motivation this EIP shouldn't be tied to EIP-1559.